tensor2tensor/rl/collect.py

```diff
@@ -112,15 +120,20 @@ def define_collect(hparams, scope, eval_phase,
     cumulative_rewards = tf.get_variable("cumulative_rewards", len(batch_env),
                                          trainable=False)
 
+    eval_phase = tf.convert_to_tensor(eval_phase)
     should_reset_var = tf.Variable(True, trainable=False)
-
     zeros_tensor = tf.zeros(len(batch_env))
 
+  if "force_beginning_resets" in hparams:
+    force_beginning_resets = hparams.force_beginning_resets
+  else:
+    force_beginning_resets = False
```

Comment:
default value for force_beginning_resets is not consistent across the board. Can we switch to all True or all False?